lighthouse-core/gather/driver.js

```diff
@@ -596,7 +596,10 @@ class Driver {
       .then(_ => this.sendCommand('Tracing.start', tracingOpts));
   }
 
-  endTrace() {
+  /**
+   * @param {number=} pauseAfterTraceMs Wait this many milliseconds before ending the trace
+   */
+  endTrace(pauseAfterTraceMs = 500) {
```

Comment:
yah. let's do 0. 

but since you mentioned it. can you add pauseBeforeTraceEnd into the perf.json as well? :)

outside of that one, custom configs are pretty rare yeah. so far we have no evidence anyone other than us are using them.